R/integrations.R

```diff
@@ -161,3 +161,115 @@ basicSeuratProc <- function(count.matrix, vars.to.regress=NULL, verbose=TRUE, do
     umap = umap
   )
 }
+
+#' RNA velocity analysis on samples integrated with conos
+#'
+#' @description Create a list of objects to pass into gene.relative.velocity.estimates function from the velocyto.R package
+#'
+#' @param cms_list list of velocity files written out as cell.counts.matrices.rds files by running dropest with -V option
+#' @param con conos object (after creating an embedding and running leiden clustering)
+#' @param n.odgenes number of overdispersed genes to use for PCA
+#' @param verbose verbose mode
+#'
+#'
+#' @return List with cell distances, combined spliced expression matrix, combined unspliced expression matrix, combined matrix of spanning reads, cell colors for clusters and embedding (taken from conos)
+#'
+#'
+#'
+#' @export
+#'
+VelocityInfoConos <- function(cms_list, con, n.odgenes = 2e3, verbose = TRUE, min.max.cluster.average.emat=0.2, min.max.cluster.average.nmat=0.05, min.max.cluster.average.smat=0.01) {
```

Comment:
Naming style: `VelocityInfoConos` to `velocityInfoConos` and `cms_list` to `cms.list`